### PR TITLE
madhumitha html-A1,A2,A3

### PR DESCRIPTION
**Pull request by Madhu4418**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: madhumitha/circle.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            
+            transform: translate(-50%, -50%)
+        }
+
+        .dot1 {
+            height: 200px;
+            width: 200px;
```
Comment (iammadhan) [inline madhumitha/circle.html]: reuse the repetitive properties

File: madhumitha/holy_grail.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    .sidebar1,
+    .sidebar2 {
+
+        background-color: grey;
+    }
+</style>
+
+<body>
+    <div>
+        <div class="header">
```
Comment (iammadhan) [inline madhumitha/holy_grail.html]: use element header

File: madhumitha/holy_grail.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <h5>Sidebar 1</h5>
+            </div>
+            <div class="article">
+                <h5>Article</h5>
+            </div>
+            <div class="sidebar2">
+                <h5>Sidebar 2</h5>
+            </div>
+        </div>
+        <div class="footer">
```
Comment (iammadhan) [inline madhumitha/holy_grail.html]: use element footer

File: madhumitha/holy_grail.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+<body>
+    <div>
+        <div class="header">
+            <h5>Header</h5>
+        </div>
+        <div class="middle">
+            <div class="sidebar1">
+                <h5>Sidebar 1</h5>
+            </div>
+            <div class="article">
```
Comment (iammadhan) [inline madhumitha/holy_grail.html]: use article element

File: madhumitha/holy_grail.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+<html>
+<!--using inline block-->
+<head>
+
+    <title>box layout</title>
+</head>
+<style>
+    .header,
+    .footer {
+        display: block;
```
Comment (iammadhan) [inline madhumitha/holy_grail.html]: change it to header, footer

File: madhumitha/holy_grail.html [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+        width: 25%;
+        margin-top: 10px;
+        text-align: center;
+        display: inline-block;
+    }
+
+    .article {
+
+        height: 200px;
```
Comment (iammadhan) [inline madhumitha/holy_grail.html]: reuse the code

File: madhumitha/overflow.html [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+            margin-left: 20px;
+            margin-bottom: 10px;
+            margin-right: 20px;
+        }
+
+        .fits-content {
+         
+            margin-top: 20px;
+            margin-left: 20px;
+            margin-bottom: 10px;
```
Comment (iammadhan) [inline madhumitha/overflow.html]: use shorthand properties

File: madhumitha/overflow.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+        }
+
+        .hidden {
+            height: 30px;
+            border: 1px solid black;
+
+            overflow: hidden;
+            padding-left: 5px;
+           
+            margin-top: 20px;
```
Comment (iammadhan) [inline madhumitha/overflow.html]: use shorthand properties

File: madhumitha/overflow.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+            border: 1px solid black;
+        }
+
+        .scroll {
+            border: 1px solid black;
+            overflow-y: scroll;
+            height: 20px;
+            margin-top: 20px;
+            margin-left: 20px;
```
Comment (iammadhan) [inline madhumitha/overflow.html]: use shorthand properties for margin, padding

File: madhumitha/overflow.html [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+            border: 2px solid black;
+            padding: 2px;
+            width: 850px;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="main">
+        <div class="header">
```
Comment (iammadhan) [inline madhumitha/overflow.html]: use header element

File: madhumitha/overflow.html [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+            <h5 style="margin-top: 5px; margin-bottom: 5px;">Overflow Hidden-The CSS box model is the foundation of layout on the Web - each element is
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+        <div class="scroll">
+            <h5>Overflow Scroll -The CSS box model is the foundation of layout on the Web - each element is
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+        <div class="footer">
```
Comment (iammadhan) [inline madhumitha/overflow.html]: use footer element

File: madhumitha/overflow.html [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+</head>
+
+<body>
+    <div class="main">
+        <div class="header">
+
+            <h5 style="position: relative; left:8px; bottom:8px;">Header</h5>
+        </div>
+        <div class="fits-content">
+            <h5 style="margin-top: 5px; margin-bottom: 5px;">Fits the content - The CSS box model is the foundation of layout on the Web - each element is
```
Comment (iammadhan) [inline madhumitha/overflow.html]: why inline style?

File: madhumitha/overflow.html [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+
+        <div class="hidden">
+            <h5 style="margin-top: 5px; margin-bottom: 5px;">Overflow Hidden-The CSS box model is the foundation of layout on the Web - each element is
```
Comment (iammadhan) [inline madhumitha/overflow.html]: change it to external style sheet

File: madhumitha/overflow.html [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+        <div class="scroll">
+            <h5>Overflow Scroll -The CSS box model is the foundation of layout on the Web - each element is
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+        <div class="footer">
+            <h5 style="position: relative; left:8px; bottom:8px;">Footer</h5>
```
Comment (iammadhan) [inline madhumitha/overflow.html]: move to external style sheet

File: madhumitha/circle.html [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+        .dot3 {
+            top: 100px;
+            left: 40px;
+            background-color: rgb(199, 11, 105);
+        }
+    </style>
+</head>
+
+<body>
+    <div class="circles main common">
```
Comment (iammadhan) [inline madhumitha/circle.html:41]: why these many classes?

File: madhumitha/overflow.html [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+        <div class="scroll">
+            <h5>Overflow Scroll -The CSS box model is the foundation of layout on the Web - each element is
+                represented as a rectangular box, with the box's content, padding, border, and margin built up
+                around one another like the layers of an onion.</h5>
+        </div>
+        <footer>
+            <h5 class="footerText">Footer</h5>
```
Comment (iammadhan) [inline madhumitha/overflow.html:38]: what is the need for footerText class?

File: madhumitha/overflow.html [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+    <title>overflow</title>
+    <link rel="stylesheet" href="overflow.css" type="text/css">
+   
+</head>
+
+<body>
+    <div class="main">
+        <header>
+
+            <h5 class="headerText">Header</h5>
```
Comment (iammadhan) [inline madhumitha/overflow.html:15]: what is the need for headerText class?

**Review by iammadhan — COMMENTED** [review]
Verified

**Pull request merged**